.storybook/config.js

```diff
@@ -1,6 +1,5 @@
 import { addDecorator, addParameters, configure } from '@storybook/html';
 import { INITIAL_VIEWPORTS } from '@storybook/addon-viewport';
-import centered from '@storybook/addon-centered/html';
```

Comment:
Removed as confusion to patterns that centered themselves.

Reply:
I've found the way of disabling `centered` on a per-story basis, see https://github.com/libero/storybook/pull/30/files#r349164152.

For this to work you'll need to revert the changes in fa8b5e9 (and then afterwards, to fix up your local environment, run `make install && make build` to get the node module back into your local environment).